node/src/chain_spec.rs

```diff
@@ -265,7 +265,7 @@ fn generate_genesis(
 				.map(|(acc, aura)| {
 					(
 						acc.clone(),                                      // account id
-						acc.clone(),                                      // validator id
+						acc,                                              // validator id
```

Comment:
It's clippy who discovered all these clone issues?

Reply:
> It's clippy who discovered all these clone issues?

yep